phoenix-core/src/it/java/org/apache/phoenix/end2end/ViewTTLIT.java

```diff
@@ -391,7 +402,1390 @@ public void testViewTTLWithAlterView() throws Exception {
         // Since the VIEW_TTL property values are not being overriden, we expect the TTL value to be different from the global view.
         assertSyscatHaveViewTTLRelatedColumns(tenantId, schemaName, tenantViewName, PTableType.VIEW.getSerializedValue(), 1000);
         assertSyscatHaveViewTTLRelatedColumns(tenantId, schemaName, indexOnTenantViewName, PTableType.INDEX.getSerializedValue(), 1000);
+    }
+
+    @Test
+    public void testViewTTLForLevelTwoViewWithNoIndexes() throws Exception {
+        long startTime = System.currentTimeMillis();
+
+        // Define the test schema.
+        // 1. Table with default columns => (ORG_ID, KP, COL1, COL2, COL3), PK => (ORG_ID, KP)
```

Comment:
what do you mean by "default" columns?

Reply:
Default columns as defined in PhoenixTestBuilder.DDLDefaults, will remove the "default" reference from the doc lines, as it is confusing folks that are reading it.